ajax/libs/WhitestormJS/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "WhitestormJS",
+  "filename": "whitestorm.js",
+  "version": "1.2.6",
+  "homepage": "https://whsjs.readme.io/docs",
+  "author": "sasha240100 <siteprogcom@gmail.com>",
```

Comment:
@pvnr0082t I wrote this based on [bower.json](https://unpkg.com/whs@1.2.6/bower.json). @sasha240100 Could you please tell me which one is correct or maybe I should wrote both? Thanks!